src/libostree/ostree-repo.h

```diff
@@ -808,6 +808,7 @@ typedef enum {
   OSTREE_REPO_CHECKOUT_OVERWRITE_NONE = 0,
   OSTREE_REPO_CHECKOUT_OVERWRITE_UNION_FILES = 1,
   OSTREE_REPO_CHECKOUT_OVERWRITE_ADD_FILES = 2, /* Since: 2017.3 */
+  OSTREE_REPO_CHECKOUT_OVERWRITE_DISJOINT_UNION_FILES = 3 /* Since: 2017.11 */
```

Comment:
Let's also add a docstring to the comment block just above this enum!

Reply:
yup, thanks for that, did not notice that at all :p,  and sorry for the late response,... as today was a intern lunch happening ( farewell to the old interns because it's their last day :'( ), pushed new one ⬆️